src/ja/common/proxy/ssh.py

```diff
@@ -1,6 +1,99 @@
+from typing import Dict
+
+from ja.common.message.base import Command, Response
+from ja.common.config import Config
+
+
 class SSHConnection:
     """
-    Establishes an SSH connection to a remote component. Writes Command objects
-    the stdin of Remote objects. Then reads a Response object from stdout of
-    the Remote object. Message objects are read/written as YAML strings.
+    Establishes an SSH connection to a Remote object. Writes Command objects to
+    the stdin of said Remote objects. Then reads a Response object from stdout
+    of the Remote objects. Message objects are read/written as YAML strings.
+    Uses paramiko as the backend for establishing an ssh connection.
     """
+    def __init__(self, ssh_config: "SSHConfig", remote: str):
+        """!
+        Creates a new SSHConnection object. Arguments for establishing the
+        actual ssh connection are packaged in @ssh_config. If no credentials
+        are provided and no suitable keys were found automatically the
+        constructor prompts the user for input.
+        @param ssh_config: Config for paramiko.
+        @param remote: The name of the Remote to execute on the host.
+        """
+
+    def send(self, command: Command) -> Response:
+        """!
+        Sends a Command to the Remote on the host. Automatically attaches
+        the username specified in ssh_config to the Command.
+        @param command: The Command to sent to the Remote.
+        @return: The Response received from the Remote.
+        """
+
+    def close(self, ) -> None:
```

Comment:
missing argument or why is there a comma here?

Reply:
I'm not sure.
I think it was added automatically by my IDE.